### DISCUSSION TITLE
怎么在官网创建model

### DISCUSSION THREAD

**Discussion by Mrgorlay** [cvat-ai/cvat#4019] [Q&A] [answered]
怎么在cvat.org创建自己的model

**Comment by nmanovic** [comment]
It is impossible at the moment.

**Reply by Mrgorlay** [reply]
I used the source code of CVAT. The platform built in the liunx environment did not show model and analysis. How to deal with this. thanks

**Comment by nmanovic** [comment, marked as answer]
https://openvinotoolkit.github.io/cvat/docs/administration/advanced/installation_automatic_annotation/

**Reply by Mrgorlay** [reply]
thanks
